bigbone/src/test/kotlin/social/bigbone/testtool/MockClient.kt

```diff
@@ -94,15 +98,19 @@ object MockClient {
             )
             .build()
 
-        every { clientMock.delete(ofType<String>(), any()) } returns response
-        every { clientMock.get(ofType<String>(), isNull(inverse = false)) } returns response
-        every { clientMock.get(ofType<String>(), any()) } returns response
-        every { clientMock.patch(ofType<String>(), any()) } returns response
-        every { clientMock.post(ofType<String>(), any()) } returns response
-        every { clientMock.post(ofType<String>(), any(), any()) } returns response
-        every { clientMock.postRequestBody(ofType<String>(), any()) } returns response
-        every { clientMock.put(ofType<String>(), any()) } returns response
-        every { clientMock.performAction(ofType<String>(), any()) } throws BigBoneRequestException("mock")
+        every { clientMock.delete(any<String>(), any<Parameters>()) } returns response
+        every { clientMock.get(any<String>(), any<Parameters>()) } returns response
+        every { clientMock.patch(any<String>(), any<Parameters>()) } returns response
+        every { clientMock.post(any<String>(), any<Parameters>(), any<Boolean>()) } returns response
+        every { clientMock.postRequestBody(any<String>(), any<RequestBody>()) } returns response
+        every { clientMock.put(any<String>(), any<Parameters>()) } returns response
```

Comment:
I’ve replaced this to better handle nullability of some of the variables. Using `any<T>()` ensures matches for null and non-null of type T.